tests/unittests/CoreGraphics.drawing/CGContextBlendModeTests.cpp

```diff
@@ -131,7 +131,8 @@ static void __drawRightStackPink(CGContextRef context, CGRect bounds) {
     CGContextRestoreGState(context);
 }
 
-class CGContextBlendMode : public WhiteBackgroundTest<>, public ::testing::WithParamInterface<::testing::tuple<bool, CGNamedBlendMode>> {
+class CGContextBlendMode : public WhiteBackgroundTest<PixelByPixelImageComparator<PixelComparisonModeDifferenceLimen<100>>>,
```

Comment:
_how far_ beyond our threshold? related: do we want to be even more nice and change our difference images to reflect strength of difference between the two images? (possibly just change color coding to be abs(expected.r - actual.r) etc etc